plugins/check_curl.c

```diff
@@ -297,6 +297,7 @@ main (int argc, char **argv)
 
 int verify_callback(int preverify_ok, X509_STORE_CTX *x509_ctx)
 {
+	(void) preverify_ok;
```

Comment:
i would prefer using `__attribute__ ( (unused))` instead.

Reply:
that's a gcc extension?